completions/cryptsetup

```diff
@@ -11,6 +11,50 @@ _cryptsetup_device()
     _filedir
 }
 
+_cryptsetup_action()
+{
+    local ret IFS=$' \t\n'
+    _comp_dequote "${1-}" || return 1
+    local cmd=${ret:-cryptsetup}
+    local actions=$(
+        {
+            LC_ALL=C "$cmd" --help 2>&1 |
+                sed -n '/^<action> is one of:/,/^[^[:space:]]/s/^[[:space:]]\{1,\}\([^[:space:]]\{1,\}\).*/\1/p'
+            LC_ALL=C man cryptsetup 2>&1 |
+                awk '/^[[:space:]]+[[:alnum:]_]+([[:space:]]+(-[^[:space:].]+|<[^<>]+>|\[[^][]+\]|or))*$/ {print $1}'
+        } | sort -u
+    )
+
+    if [[ ! $actions ]]; then
+        # The fallback action list is extracted from the following source:
+        # https://gitlab.com/cryptsetup/cryptsetup/-/blob/main/src/cryptsetup.c#L3154-3208 (search for "Handle aliases")
+        # https://gitlab.com/cryptsetup/cryptsetup/-/blob/main/src/cryptsetup.c#L2831-2867 (search for "struct action_type")
+        # https://gitlab.com/cryptsetup/cryptsetup/-/blob/main/src/cryptsetup_args.h#L28-53 (see the macros "*_ACTION")
+        actions='benchmark bitlkClose bitlkDump bitlkOpen close config convert
+            create erase isLuks loopaesClose loopaesOpen luksAddKey
+            luksChangeKey luksClose luksConfig luksConvertKey luksDump
+            luksErase luksFormat luksHeaderBackup luksHeaderRestore
+            luksKillSlot luksOpen luksRemoveKey luksResume luksSuspend luksUUID
+            open plainClose plainOpen reencrypt refresh remove repair resize
+            status tcryptClose tcryptDump tcryptOpen token'
+
+        # We attempt to filter the supported actions by the strings in the binary.
```

Comment:
If we decide to keep the fallback, I suggest we don't do this. It's not something we do elsewhere, and frankly I think it's too much for a corner case here.

Then again, if there is a bunch of other cases we feel we should do this in other completions, then a separate helper function for doing it would be in order, in another PR. Not quite sure myself, but open to opinions.